src/compass/utils/validate_product.py

```diff
@@ -0,0 +1,333 @@
+'''
```

Comment:
What do you think about putting a hashbang, so we can use it as a script? In that case, the permission to execute needs to be added.
```suggestion
#!/usr/bin/env python
'''
```

Reply:
Typically COMPASS would need to be installed to run this, so wouldn't it be easier to script with the sample commands I provided in the PR description?